### ISSUE TITLE
No traffic on anonymous downloads

### ISSUE THREAD

**Issue by ichorid** [Tribler/tribler#5175]
Latest `devel` and 7.5.0-exp1. When setting anonymity mode to **one-hop**, the download traffic stops, no peers. Also, can't get torrent information from DHT at all.

It important to note that two- and three-hop downloads work, albeit take more than a minute to start.

**Comment by ichorid** [comment]
@egbertbouman , could you please look at this?

**Comment by egbertbouman** [comment]
@ichorid Turn out that this is happening because libtorrent is now started before the TunnelCommunity (due to https://github.com/Tribler/tribler/commit/c00cfacac2c62cdc87987422ee18aeb44d1ff28c). I'll fix it in a bit.

**Comment by ichorid** [comment]
> @ichorid Turn out that this is happening because libtorrent is now started before the TunnelCommunity (due to [c00cfac](https://github.com/Tribler/tribler/commit/c00cfacac2c62cdc87987422ee18aeb44d1ff28c)). I'll fix it in a bit.

Oops, that was my fault, after all. Anyways, I would expect Tunnels community to depend on Libtorrent, not the other way around.

**Comment by egbertbouman** [comment]
@ichorid In that case, feel free to fix in a way that you feel does make sense.

**Comment by ichorid** [comment]
fixed in #5184

**Comment by michaelsmoody** [comment]
Good afternoon. First, let me say thank you, as I strongly prefer Tribler to most other options available on macOS.

I've looked for other issues, and this appears to match closest. This appears to still be happening in 7.5.0-rc5. I can't download well seeded files such as an Ubuntu image, or Kali Nethunter unless disabling anonymity entirely. Downloads that worked a few weeks ago (and were paused, upgrade, resumed) no longer download.

Any guidance about where to look is appreciated (Tor works, so it doesn't appear to be a network blocking issue).

Thanks again.

**Comment by devos50** [comment]
Hi and thank you for using Tribler!

I can't immediately tell what would be causing this issue. It might be that exit nodes are overloaded and that you therefore do not get a circuit assigned. What's your token balance? Also, is the anonymous downloading failing when trying to download it at another time during the day?

If the torrent is Ubuntu content, could you share the `.torrent` file (or a link to that file) with us, so we can try to reproduce and debug this issue?

**Comment by michaelsmoody** [comment]
https://images.offensive-security.com/nethunter/nethunter-hammerhead-marshmallow-kalifs-full-2020.1.zip.torrent?_ga=2.104490624.1481424012.1588706084-464733534.1587774066 is an example of a torrent that is well seeded that wouldn't download anymore unless I disabled anonymity. To double check, I added the torrent again, and it is still "stuck" until disabling anonymity.

My token balance is -12.8GB, though no longer how long I leave Tribler running, it never goes up, only down. Perhaps that issue is related?
